### DISCUSSION TITLE
Frameskipping for low-end devices

### DISCUSSION THREAD

**Discussion by gunir** [mpv-player/mpv#14716] [Q&A]
### Expected behavior of the wanted feature

Hi, I've tried to search mpv.io and it happened that there's no such feature.

For low-end devices, playing 60fps videos making their computer lag and overheat, and system becomes slower, it would be great to able to skip frame to lower fps to make it watchable and less system overhead.

**Comment by Andarwinux** [comment]
`vf=fps=30`
But if the bottleneck is decoding, this won't help.

**Comment by gunir** [comment]
> But if the bottleneck is decoding, this won't help.

Yeah, this is the hard part, would be great if there's a way to skip decoding too, that's what I expected and I've tried vf=fps, not really helpful for heavy format like AV1

**Reply by hooke007** [reply]
> not really helpful for heavy format like AV1

Only if av1 could provide some acc-workaround like avc/hevc https://mpv.io/manual/master/#options-vd-lavc-skiploopfilter

**Comment by kasper93** [comment]
> not really helpful for heavy format like AV1

If your hardware is not capable of decoding given format in realtime there is nothing you can do.

For low end devices you will get best results with `--vo=gpu-next --profile=fast --hwdec` or `--vo=dmabuf-wayland --hwdec`.

**Reply by gunir** [reply]
I'm still wondering is skipping frame from decoding possible or it must be supported by the format ?

**Comment by geextahslex** [comment]
probably the best thing you can do is to lower the resolution
press `I` and then `2` and check if theres is something active like debanding or dithering or other stuff. maybe you can disable it

**Reply by gunir** [reply]
There's framedrop=vo+decoder, this does drop frames in some edge cases but there's no way to forcefully drop frame from decoder to complement this feature.

https://github.com/mpv-player/mpv/issues/4995
